flasc/data_processing/time_operations.py

```diff
@@ -56,21 +56,21 @@ def df_movingaverage(
     # Carry out the mean calculations
     df_regular = (
         df[cols_regular]  # Select only non-angular columns
-        .rolling(window_width, center=center, axis=0, min_periods=min_periods)
```

Comment:
was axis formerly implicit and now has to be explicit?

Reply:
`axis` used to default to 0, but you were allowed to apply the rolling average along the column axis if you liked. This behavior has been deprecated; in future, `rolling` will not accept an axis and will _only_ apply along the index axis. To apply it across columns, the user will have to first transpose the dataframe and then transpose it back.

https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.rolling.html